taskcluster/translations_taskgraph/transforms/training_continuation.py

```diff
@@ -0,0 +1,94 @@
+from taskgraph.transforms.base import TransformSequence
+from urllib.parse import urljoin
+import os
+
+CONTINUE_TRAINING_ARTIFACTS = (
+    "devset.out",
+    "model.npz",
+    "model.npz.best-bleu-detok.npz",
+    "model.npz.best-bleu-detok.npz.decoder.yml",
+    "model.npz.best-ce-mean-words.npz",
+    "model.npz.best-ce-mean-words.npz.decoder.yml",
+    "model.npz.best-chrf.npz",
+    "model.npz.best-chrf.npz.decoder.yml",
+    "model.npz.decoder.yml",
+    "model.npz.optimizer.npz",
+    "model.npz.progress.yml",
+    "model.npz.yml",
+    "train.log",
+    "valid.log",
+    "vocab.spm",
+)
+
+INITIALIZE_MODEL_ARTIFACTS = (
+    "model.npz.best-bleu-detok.npz",
+    "model.npz.best-ce-mean-words.npz",
+    "model.npz.best-chrf.npz",
+)
+
+
+def get_artifact_mount(url, directory, artifact_name):
+    normalized_url = f"{url}/" if not url.endswith("/") else url
+    artifact_url = urljoin(normalized_url, artifact_name)
+    return {
+        "content": {
+            "url": artifact_url,
+        },
+        "file": os.path.join(directory, artifact_name),
+    }
+
+
+def get_artifact_mounts(urls, directory, artifact_names):
+    for url in urls:
+        artifact_mounts = []
+        for artifact_name in artifact_names:
+            artifact_mounts.append(get_artifact_mount(url, directory, artifact_name))
+        yield artifact_mounts
+
+
+transforms = TransformSequence()
+
+
+def validate_pretrained_models(config):
+    pretrained_models = config.params["training_config"]["experiment"].get("pretrained-models", {})
```

Comment:
suggestion (non-blocking): This looks like it's purely validating parameters. If so, this is probably better to do when we first see them in `train.py`.